tasks_downstream.py

```diff
@@ -416,7 +432,16 @@ def start(c, detach=True, debugpy=False):
         if detach:
             cmd += " --detach"
         with c.cd(str(PROJECT_ROOT)):
-            c.run(cmd, env=dict(UID_ENV, DOODBA_DEBUGPY_ENABLE=str(int(debugpy))))
+            result = c.run(
+                cmd,
+                pty=True,
+                env=dict(
+                    UID_ENV,
+                    DOODBA_DEBUGPY_ENABLE=str(int(debugpy)),
+                ),
+            )
+            if not ("Recreating" or "Starting" in result.stdout):
```

Comment:
This won't work as you expect:

```py
>>> "Recreating" or "Starting" in "hello"
'Recreating'
>>> bool("Recreating" or "Starting" in "hello")
True
```

Reply:
Oh of course :man_facepalming: 
Sorry, MB. Changes made!